api/src/pcapi/core/subscription/dms/api.py

```diff
@@ -34,41 +19,35 @@ def handle_dms_state(
     state: api_dms.GraphQLApplicationStates,
 ) -> None:
 
-    if state == api_dms.GraphQLApplicationStates.accepted:
+    logs_extra = {
+        "application_id": application_id,
+        "procedure_id": procedure_id,
+        "user_email": user.email,
+    }
 
-        logger.info(
-            "DMS Application accepted.",
-            extra={
-                "application_id": application_id,
-                "procedure_id": procedure_id,
-                "user_email": user.email,
-            },
-        )
-    elif state in (
-        api_dms.GraphQLApplicationStates.draft,
-        api_dms.GraphQLApplicationStates.on_going,
-    ):
+    current_fraud_check = fraud_dms_api.get_fraud_check(user, application_id)
+    if current_fraud_check is None:
+        # if we miss any webhook event, enforce the creation of the fraud_check
```

Comment:
create a fraud_check whatever the status is because we may have missed a webhook event

Reply:
done